### PR TITLE
`cycle retro` --> `cycle reflect`

### LINKED ISSUE #22: `/cycle retro` --> `/cycle reflect`
See also: https://github.com/LearnersGuild/game/issues/136

### PR DESCRIPTION
**Pull request by jeffreywescott**
Fixes #22 

Related to https://github.com/LearnersGuild/game/issues/136

RETROSPECTIVE state is now REFLECTION

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@learnersguild/game-cli",
-  "version": "0.7.3",
+  "version": "0.7.4",
```
Comment (bundacia) [inline package.json]: technically this should be a major version change since it's backwards incompatible.

**Comment by bundacia** [comment]
HISTORY file needs to be updated.

**Comment by bundacia** [comment]
Assigned back to you @jeffreywescott. Two minor comments. Fix if you agree and then feel free to merge on your own.

File: package.json (hunk 1 above)
Comment (jeffreywescott) [inline package.json]: Nice catch!

**Pull request merged**